### ISSUE TITLE
cmake does not find package CapnProto

### ISSUE THREAD

**Issue by mhk** [capnproto/capnproto#589]
Hello,

I am having trouble to integrate capnproto into my cmake project. I am getting `Could not find a package configuration file provided by "CapnProto" with` similar (or identical?) to [Issue 630](https://github.com/numenta/nupic.core/issues/630). The provided instructions do not work for me, could you please clarify.

## Steps to reproduce
1. Start with a clean ubuntu 16.04 installation (optional)
2. Install required tools
    ```
    apt install -y git autoconf automake cmake libtool g++
    ```
3. Download capnproto:
    ```
    git clone https://github.com/capnproto/capnproto.git
    cd capnproto/c++
    ```
4. Either compile from source:
    ```
    autoreconf -i && ./configure && make -j6 check && sudo make install
    ```
    **or** install with package manager
    ``` 
    apt install -y capnproto
    ```
5. Build samples
    ```
    mkdir ../build-samples
    cd ../build-samples
    cmake ../c++/samples
    cmake --build .
    ./addressbook write | ./addressbook read
    ```

### Expected result:
````
Alice: alice@example.com
  mobile phone: 555-1212
  student at: MIT
Bob: bob@example.com
  home phone: 555-4567
  work phone: 555-7654
  unemployed
````

### Actual result:
```
-- The CXX compiler identification is GNU 5.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:22 (find_package):
  Could not find a package configuration file provided by "CapnProto" with
  any of the following names:

    CapnProtoConfig.cmake
    capnproto-config.cmake

  Add the installation prefix of "CapnProto" to CMAKE_PREFIX_PATH or set
  "CapnProto_DIR" to a directory containing one of the above files.  If
  "CapnProto" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/aya/capnproto/build-samples/CMakeFiles/CMakeOutput.log".
```

**Comment by harrishancock** [comment]
Hi @mhk, this is actually a duplicate of #523. I have a PR to fix it at #575 that should land soon, though it will take a while for it to trickle down to distro package management systems.

In the meantime, you can work around the problem by compiling from source using CMake, maybe something like:

```sh
cd capnproto/c++
mkdir build
cd build
cmake .. -DCMAKE_INSTALL_PREFIX=/your/install/prefix
make install -j6

cd ../samples
mkdir build
cd build
cmake .. -DCMAKE_PREFIX_PATH=/your/install/prefix
make -j6
```

**Comment by harrishancock** [comment]
@kentonv just merged #575, so the fix should be on master. @mhk, want to try it out and close the issue if it works?

**Comment by mhk** [comment]
Thank you @harrishancock. It's working now as expected.
